internal/driver/driver_test.go

```diff
@@ -124,15 +121,16 @@ func TestParse(t *testing.T) {
 			if flags[0] == "topproto" {
 				f.bools["proto"] = false
 				f.bools["topproto"] = true
+				f.bools["addresses"] = true
 			}
 
 			// First pprof invocation to save the profile into a profile.proto.
-			o1 := setDefaults(nil)
-			o1.Flagset = f
-			o1.Fetch = testFetcher{}
-			o1.Sym = testSymbolizer{}
-			o1.UI = testUI
-			if err := PProf(o1); err != nil {
+			o := setDefaults(nil)
```

Comment:
Could you keep the original variable names? Sorry, I probably wasn't clear previously when I asked to use re-assignment for the `f` variable - I don't really care much one way or another, I am just suggesting we limit unnecessary styling changes so that the diff is more telling.

Reply:
Done.